post-types/templates/page-profiles-list-page.php

```diff
@@ -16,7 +16,17 @@
         <?php
           // Display Total list
           $show_total_value = "";
-          $array_map_profile = array_map(function($value){return array_key_exists('map_id', $value) ? $value['map_id'] : "";}, $profiles);
+          $array_map_profile = array();
```

Comment:
@Huyeng I have added the creation of the **array_map_profile** and **id** variables outside the **if** clause. Variables should be initialized in the proper scope.